session-01/validators.js

```diff
@@ -17,7 +28,23 @@ function validUsername(username) {
   - password must contain at least 1 letter, 1 number, and 1 special character
 */
 function validPassword(password) {
-  return;
+  let valid = false;
+  const passwordLetterFormat = /[a-zA-Z]/;
+  const passwordNumberFormat = /[0-9]/;
+  const specialFormat = /[!@#$%^&*()?.,;{}|/\-=]/;
+  let passCount = false;
+  if(password.match(passwordLetterFormat) && 
+    password.match(specialFormat) && 
+    password.match(passwordNumberFormat)){
+      passCount = true;
+  }
+
+  if(password.length >= 10 && password.length <= 64){
+    if(passCount){
+      valid = true;
+    }
+  }
```

Comment:
Consider using guard clauses that return false without having to recheck passcount, like in this https://github.com/CUNYTechPrep/js-summer-prep-2024/blob/main/session-01/validators.solution.js#L10-L16